R/hiper_glm.R

```diff
@@ -2,21 +2,24 @@
 hiper_glm <- function(design, outcome, model='linear', option = list()){
 
   ## Check validity
-  supported_models <- c('linear')
-  if(!(model %in%  supported_models)){stop(sprintf("The model %s is not supported.",
-                                                   model))
+  supported_models <- c("linear", "logit")
+  if(!(model %in%  supported_models)){
+    stop(sprintf("The model %s is not supported.", model))
   }
 
 
   if (is.null(option$mle_solver)) {
```

Comment:
may have mentioned this in hw1, but you may want some logic here to error if an invalid solver is provided, since right now if someone specifies "gradient descent", itll do newton raphson and they won't know.